kyaml/openapi/scripts/fetchSchemaFromCluster.sh

```diff
@@ -8,8 +8,13 @@ VERSION=$1
 
 cp $HOME/.kube/config /tmp/kubeconfig.txt | true
 $MYGOBIN/kind create cluster --image kindest/node:$VERSION --name=getopenapidata
-$MYGOBIN/kustomize openapi fetch > /tmp/new_swagger.json
+
+# TODO (natasha41575) Add a `kustomize openapi fetch --proto` option
+kubectl proxy &
+sleep 5
```

Comment:
I needed to add this delay because sometimes the `curl` timed out before the kubectl proxy was ready